nnpdf_data/nnpdf_data/commondata/ATLAS_Z0_7TEV_49FB/metadata.yaml

```
@@ -1,61 +1,52 @@
 setname: ATLAS_Z0_7TEV_49FB
-version: 1
-version_comment: Port of old commondata
+version: 2
+version_comment: Reimplementation of old commondata
 nnpdf_metadata:
   nnpdf31_process: DY NC
   experiment: ATLAS
 arXiv:
   url: https://arxiv.org/abs/1305.4192
   journal: Phys.Lett. B725 (2013) 223
 iNSPIRE:
-  url: ''
+  url: 'https://inspirehep.net/literature/1234228'
 hepdata:
-  url: 10.17182/hepdata.61422.v1/t1
+  url: https://www.hepdata.net/record/ins1234228
   version: -1
 implemented_observables:
 - observable_name: HIMASS
   observable:
     description: Drell-Yan Mass Distribution
     label: ATLAS HM DY 7 TeV
     units: ''
-  process_type: EWK_MLL
-  tables: []
+  process_type: "DY_MLL" 
+  tables: [1]
   npoints: []
   ndata: 13
   plotting:
-    kinematics_override: ewk_mll_sqrt_scale
+    kinematics_override: identity
     dataset_label: ATLAS HM DY 7 TeV
     y_label: $d\sigma_{Z/\gamma^{*}}/dM_{ll}$ (fb)
     y_scale: log
-    plot_x: k2
+    plot_x: M_ll 
   kinematic_coverage:
-  - k1
-  - k2
-  - k3
+  - M_ll
   kinematics:
     variables:
-      k1:
-        description: Variable k1
-        label: k1
-        units: ''
-      k2:
-        description: Variable k2
-        label: k2
-        units: ''
-      k3:
-        description: Variable k3
-        label: k3
-        units: ''
-    file: kinematics_HIMASS.yaml
+      M_ll:
+        description: Mass of lepton pair
+        label: M_ll
+        units: 'GeV'  
+    file: kinematics.yaml
   theory:
     conversion_factor: 1000.0
     operation: 'null'
     FK_tables:
     - - ATLAS_DY_7TEV_49FB_HIMASS
-  data_uncertainties: []
-  variants:
-    legacy:
-      data_uncertainties:
-      - uncertainties_legacy_HIMASS.yaml
-  data_central: data_legacy_HIMASS.yaml
+  data_uncertainties:
+    - uncertainties.yaml
+  # variants:
+  #   legacy:
+  #     data_uncertainties:
+  #     - uncertainties_legacy_HIMASS.yaml
```

```suggestion
  variants:
    legacy:
      data_uncertainties:
      - uncertainties.yaml
```

Please don't remove the legacy variant yet, just point to the new uncertainties if they are exactly the same.
(and, since they are the same, please remove `uncertainties_legacy_HIMASS.yaml`)

---

Ok, I've just pushed that change